lib/classes/Service.js

```diff
@@ -33,6 +33,8 @@ class Service {
   load(opts) {
     const that = this;
     const options = opts || {};
+    options.stage = options.stage || options.s;
```

Comment:
I have no idea about this code but IMHO Service class/object should have no idea about cli params. It should get already parsed/processed options.